src/get-started/java-binding.md

```diff
@@ -553,6 +553,10 @@ try (TestKit testKit = TestKit testKit = TestKit.builder()
 }
 ```
 
+Here and elsewhere, `ARTIFACT_ID`, `ARTIFACT_FILENAME`, and other constants are
```

Comment:
@Maria-Nosyk could you please check this paragraph?

Reply:
I think it doesn't make sense in such formulation (how system properties are connected to these properties, where the values come from?).

*Things in brackets can be omitted if it is clear without them:*

In testkit code examples, `ARTIFACT_ID`, `ARTIFACT_FILENAME`, and `ARTIFACTS_DIRECTORY` correspond to constants defining the corresponding properties [of the artifact and the environment]. As the values [of these properties] are usually defined in the build configuration, it is recommended to pass them to the test from there (e.g., via system properties set in `maven-failsafe-plugin`).

Reply:
I'd consider if it makes sense saying 'and the other', because they clearly are not _defined_ in the build configuration and shall not be passed from there.

Reply:
In testkit code examples, ARTIFACT_ID, ARTIFACT_FILENAME, and ARTIFACTS_DIRECTORY correspond to constants defining the corresponding properties of the artifact and the environment. As the values of these properties are usually defined in the build configuration, it is recommended to pass them to the test from the build configuration (e.g., via system properties set in maven-failsafe-plugin).